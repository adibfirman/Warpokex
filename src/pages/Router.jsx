import React, { Suspense } from 'react'
import history from 'pages/history'
import { Router, Route } from 'react-router-dom'
import Loading from 'components/Loading'

const Home = () => <p>Home</p> 

export default function Routers() {
  return (
    <Router history={history}>
      <Suspense fallback={<Loading />}>
        <Route key="homepage" path="/" component={Home} exact />
      </Suspense>
    </Router>
  )
}
