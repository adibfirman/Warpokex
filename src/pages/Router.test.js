import React from 'react'
import Router from './Router'

test('Should Render router correctly', () => {
  const { container } = render(<Router />)
  expect(container).toMatchSnapshot()
})
