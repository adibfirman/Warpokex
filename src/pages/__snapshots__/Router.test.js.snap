// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Render router correctly 1`] = `
<div>
  <div
    class="sc-bdVaJa HAhlL"
    style="position: absolute; top: 50vh; bottom: 50vh; transform: translateX(20rem);"
  />
</div>
`;
